source/webapp/sites/all/modules/custom/checkbook_widget_redesign/checkbook_business_layer/checkbook_services/budget/BudgetWidgetService.php

```diff
@@ -102,6 +102,6 @@ public function adjustParameters($parameters, $urlPath) {
     }
 
     public function getWidgetFooterUrl($parameters) {
-        return BudgetUrlService::getFooterUrl($parameters,$this->getLegacyNodeId());
```

Comment:
$parameters was removed from function definition, as it is not used anywhere inside the function. This is the only place that this function is called. Hence removed $parameters.

Reply:
@nranevj, can  you please make sure the is change does not impact widget footer url on the UI also?

Reply:
@sgade-reisys this change does not impact the widget footer URL on the UI. Have confirmed the same by comparing local URLs with production ones.